File: Training/URI/categoria-Iniciante/java/1183.java

import java.util.Scanner;;

public class Main {
	public static void main(String[] args) {
		Scanner scanner = new Scanner(System.in);
		double[][] matriz = new double[12][12];
		String op = scanner.next();
		double soma = 0;
		int count = 0;
		for(int i = 0; i < 12; i++) {
			for(int j = 0; j < 12; j++) {
				matriz[i][j] = scanner.nextDouble();
				if(j > i) {
					soma += matriz[i][j];
					count ++;
				}
			}
		}

		if(op.equals("S")) {
			System.out.printf("%.1f\n",soma);
		} else {
			System.out.printf("%.1f\n", soma / count);
		}
	}
}

File: Training/URI/categoria-Iniciante/C/1183.c
#include<stdio.h>
main() {
	float matriz[12][12];
	float soma = 0;
	int count = 0;
	char op;
	fflush(stdin);
	scanf("%c",&op);
	int i,j;
	for(i = 0; i < 12; i++) {
		for(j = 0; j < 12; j++) {
			scanf("%f",&matriz[i][j]);
		if(j > i) {
				soma += matriz[i][j];
				count++;
			}
		}
	}
	float media;
	if(op == 'S') {
		printf("%.1f\n", soma);
	} else if(op == 'M'){
		media = soma/12;
		printf("%.1f\n", soma/count);
	}
}
	
